core/variant/variant_call.cpp

```diff
@@ -1928,18 +1930,28 @@ static void _register_variant_builtin_methods() {
 	bind_method(Basis, inverse, sarray(), varray());
 	bind_method(Basis, transposed, sarray(), varray());
 	bind_method(Basis, orthonormalized, sarray(), varray());
+	bind_method(Basis, orthogonalized, sarray(), varray());
 	bind_method(Basis, determinant, sarray(), varray());
 	bind_methodv(Basis, rotated, static_cast<Basis (Basis::*)(const Vector3 &, real_t) const>(&Basis::rotated), sarray("axis", "angle"), varray());
 	bind_method(Basis, scaled, sarray("scale"), varray());
+	bind_method(Basis, scaled_local, sarray("scale"), varray());
+	bind_method(Basis, scaled_orthogonal, sarray("scale"), varray());
 	bind_method(Basis, get_scale, sarray(), varray());
+	bind_method(Basis, get_scale_local, sarray(), varray());
+	bind_method(Basis, get_scale_abs, sarray(), varray());
 	bind_method(Basis, get_euler, sarray("order"), varray((int64_t)EulerOrder::YXZ));
+	bind_method(Basis, set_euler, sarray("euler", "order"), varray());
+	bind_method(Basis, set_quaternion, sarray("quaternion"), varray());
 	bind_method(Basis, tdotx, sarray("with"), varray());
 	bind_method(Basis, tdoty, sarray("with"), varray());
 	bind_method(Basis, tdotz, sarray("with"), varray());
+	bind_method(Basis, lerp, sarray("to", "weight"), varray());
```

Comment:
This would close https://github.com/godotengine/godot/issues/68491 (at least on Core's side, this would still need to be exposed in C# too).

Reply:
Yeah, I think lerp would be useful for Basis completions involving shears, and I'll look into implementing it later for C# as well.

Reply:
Exposed it in CS.